### PR TITLE
PR for Apple website clone

### PR DESCRIPTION
**Pull request by UmairAhmad125**
This PR is for Building with Backgrounds and Gradients (Apple's old website clone).
The project includes gradients as well as background images. 
The project was manually tested.
Details of the project are in the readme file.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  flex-flow: column;
+  background-image: url("../resources/background.jpg");
+  height: 540px;
+  width: 100%;
+  background-color: lightblue;
+  background-repeat: no-repeat;
+  background-position: bottom;
+
+  /* position: fixed; */
+}
```
Comment (lucasmazo32) [inline css/style.css]: - [x] Please do not set the background color unless necessary. For the development of the webpage, it helps you with the size of some icons but for the production it doesn't look professional.

- [x] Please remove all commented code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            </div>
+            <div class="bottom-pic2">
+                <img src="resources/5c.jpg" alt="iPhone 5c">
+            </div>
+            <div class="bottom-pic3">
+                <img src="resources/bday.jpg" alt="Apple Anniversary">
+            </div>
+            <div class="bottom-pic4">
+                <img src="resources/supplier.jpg" alt="Apple Supplier">
+            </div>
```
Comment (lucasmazo32) [inline index.html]: - [x] This images should be enclosed inside `<a></a>` tags. You can link to '#' if you want.

- [x] In my screen they are not properly aligned with the background. I will attatch a screenshot.

> image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved kindly implement these changes:

## Required changes:

- [x] Please enable the stickler for this repo.

- [ ] Check the comments below.

### Happy coding! :computer:

## Lucas Mazo

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Status: changes required
hello @UmairAhmad125** 

You are an inch closer to getting :+1:  for this project as it covered all the requirements except stickler validation. It seems that you forgot to fix an error addressed by the stickler. Please fix that error and submit for another review as it is mandatory. Otherwise, everything looks perfect to me.

[optional] if you want you can set `text-decoration: none` to every a tag as it is repeating four times.

Need support? Message me on slack **Suman Shrestha**
**Happy coding !!**

**Review by sumancrest0001 — APPROVED** [review]
**Status: APPROVED** 
Congratulation !!! @UmairAhmad125 for completing another project. Now, you can merge the branch with the master and submit the project completion form.
mission

**Pull request merged**